pkg/cli/out/table.go

```diff
@@ -16,7 +16,17 @@ const (
 	targetOverhead = 29
 )
 
+var noColorEnabled bool
```

Comment:
Quick suggestion too: could you rename to `isColorDisabled` instead?

A common convention is to name booleans with `is` or `has` as prefix